app/main.py

```diff
@@ -1,3 +1,70 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, distance):
+        if isinstance(distance, Distance):
+            distance.km = self.km + distance.km
+            return Distance(distance.km)
+        elif isinstance(distance, (int, float)):
+            distance = self.km + distance
+            return Distance(distance)
```

Comment:
You may be sure that all data you get is valid, and you get only - `Distance`, `int`, and `float`, so you can remove these double checks. (change in all methods this change is available)